### PR TITLE
Update Project README

### PR DESCRIPTION
**Pull request by byoms**
- Changed the primary theme of the content to revolve around our website rather than the template project (presumably unchanged since fork)
  - Retained most essential info and omitted the rest
- Other minor improvements such as better spacing to improve readability etc.
- Added useful specific info for local development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,130 +1,89 @@
...
+and run next command from root folder:
 ```bash
-  bundle install
+bundle install
 ```  
 
 To start Jekyll run:
 ```bash
-    jekyll serve -w
+jekyll serve -w --baseurl /test
```
Comment (ashwinimanoj) [inline README.md:32]: This is only for local development, could you add that as a separate part?

File: README.md [hunk 2]
```diff
@@ -1,130 +1,89 @@
-# Project Zeppelin / GDG DevFest 2014 site template
+# DevOpsDays India - Bengaluru Chapter - Website #
 
-### About
-Project Zeppelin allows you to setup awesome GDG DevFest site in 5 minutes.
+## About ##
 
-Project is built on top of [Jekyll](http://jekyllrb.com/) - simple, blog-aware, static site generator. Jekyll also happens to be the engine behind GitHub Pages, which means you can use Jekyll to host your website from GitHub’s servers for free. [Learn more about Jekyll](http://jekyllrb.com/).
+This project powers the [DevOpsDays India Website](https://devopsdaysindia.org/) (Bengaluru chapter). It uses the Project Zeppelin / GDG DevFest 2014 site template.
```
Comment (ashwinimanoj) [inline README.md:5]: Maybe you could leave a link to Jekyll as well in this line?

File: README.md [hunk 3]
```diff
@@ -1,130 +1,89 @@
...
 
 To watch changes of `.sass` files and compile it to the `.css` on a fly change property `safe: true` to `safe: false` in `_config.yml`.
 **Note: It works only on local machine, because GitHub runs Jekyll in `--save` [mode](https://help.github.com/articles/using-jekyll-with-pages/#configuration-overrides)**
 
-Learn more about Sass development from [documentation](https://github.com/gdg-x/zeppelin/wiki/Sass-development).
+Learn more about Sass development from this [documentation](https://github.com/gdg-x/zeppelin/wiki/Sass-development) by the GDG team.
 
 
-### Resource optimizations (optional)
+### Resource optimizations (optional) ###
```
Comment (ashwinimanoj) [inline README.md:49]: Did you mean to add the ### at the end? Just checking

Comment (byoms) [reply README.md:49]: Yes, just a personal preference.

File: README.md (hunk 1 above)
Comment (byoms) [reply README.md:32]: Actually this is under the local development section itself?

File: README.md (hunk 2 above)
Comment (byoms) [reply README.md:5]: Updated with new commit - [7d0d365](https://github.com/ashwinimanoj/devopsdays/pull/1/commits/7d0d36522f1ab3802ea40a16eefde3a0f551ee86)

File: README.md (hunk 1 above)
Comment (ashwinimanoj) [reply README.md:32]: Ah my bad, ignore in that case

**Pull request merged**